### PR TITLE
fix(traversing): update terminology and screenshot

### PR DESCRIPTION
**Pull request by doralchan**
Linked to https://github.com/getsentry/sentry/pull/27431

Screen Shot 2021-07-14 at 3 40 32 PM

### CODE REVIEW

File: src/docs/product/sentry-basics/tracing/event-detail.mdx [hunk 1]
```diff
@@ -68,19 +68,19 @@ The span view may be limited to one project at a time if you're on the [Team pla
 
 </Alert>
 
-#### Traversing and Nesting Transactions
```
Comment (dashed) [inline src/docs/product/sentry-basics/tracing/event-detail.mdx:71]: we might want to be explicit about "Nesting"

Comment (imatwawana) [reply src/docs/product/sentry-basics/tracing/event-detail.mdx:71]: What about "Traversing and Adding Transactions" as the heading? Does that address your concern @dashed or do you think we need to stick with the "nesting" terminology?

Comment (dashed) [reply src/docs/product/sentry-basics/tracing/event-detail.mdx:71]: I think we should be explicit that "Traversing" isn't the same as embedding/nesting/inlining/adding a transaction. I don't have a strong opinion of what the new product feature should be called.

File: src/docs/product/sentry-basics/tracing/event-detail.mdx [hunk 2]
```diff
@@ -68,19 +68,19 @@ The span view may be limited to one project at a time if you're on the [Team pla
...
+#### Traversing Transactions
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/tracing/event-detail.mdx:71]:
```suggestion
#### Traversing & Adding Transactions
```

File: src/docs/product/sentry-basics/tracing/event-detail.mdx (hunk 1 above)
Comment (imatwawana) [reply src/docs/product/sentry-basics/tracing/event-detail.mdx:71]: I've added my heading suggestion on line 71

File: src/docs/product/sentry-basics/tracing/event-detail.mdx (hunk 2 above)
Comment (imatwawana) [inline src/docs/product/sentry-basics/tracing/event-detail.mdx:71]:
```suggestion
#### Traversing & Adding Transactions
```

**Review by dashed — APPROVED** [review]

**Pull request merged**